addons/web/static/tests/core/dialog_tests.js

```diff
@@ -281,30 +293,44 @@ QUnit.module("Components", (hooks) => {
 
     QUnit.test("Interactions between multiple dialogs", async function (assert) {
         assert.expect(14);
+        serviceRegistry.add(
+            "dialog",
+            makeFakeDialogService({
+                close: (id) => {
+                    assert.step("close with id: " + id);
+                    delete parent.dialogIds[id];
+                },
+            }),
+            { force: true }
+        );
+        env = await makeTestEnv();
+        let id = 0;
+        class MyDialog extends SimpleDialog {
+            setup() {
+                super.setup();
+                this.__id = id;
+            }
+        }
         class Parent extends owl.Component {
             constructor() {
                 super(...arguments);
                 this.dialogIds = useState({});
             }
-            _onDialogClosed(id) {
-                assert.step(`dialog_id=${id}_closed`);
-                delete this.dialogIds[id];
-            }
         }
         Parent.template = owl.tags.xml`
               <div>
-                <SimpleDialog t-foreach="Object.keys(dialogIds)" t-as="dialogId" t-key="dialogId"
-                  t-on-dialog-closed="_onDialogClosed(dialogId)"
-                  />
+                <MyDialog t-foreach="Object.keys(dialogIds)" t-as="dialogId" t-key="dialogId"/>
```

Comment:
I'm wondering if those tests aren't simply irrelevant now. Maybe we should test the trio dialog/dialogService/dialogContainer together